src/data/tournament.py

```diff
@@ -232,7 +232,9 @@ def first_board_number(self) -> int:
 
     @property
     def paired_bye_points(self) -> Result:
-        return Result(self.stored_tournament.paired_bye_points) or PapiWebConfig.default_paired_bye_points
+        return Result(self.stored_tournament.paired_bye_points) \
+            if self.stored_tournament.paired_bye_points is not None \
+            else PapiWebConfig.default_paired_bye_points
```

Comment:
Formatting smell: `\` line continuations.
```suggestion
	if (points_id := self.stored_tournament.paired_bye_points) is not None:
	    return Result(points_id)
        return PapiWebConfig.default_paired_bye_points
```

Reply:
Fixed in my last commit.